Makefile

```diff
@@ -52,16 +52,16 @@ build-prod:
 
```

Comment:
This did not work for me unless I change the `generate` line to 
```
go generate pkg/registrationserver/routes.go
```
to explicitly run the file with the `generate` comment. All of my other comments on the paths may be related to this. Imo this is weird, I would expect `go generate` to pick up all `generate` comments in the source, but for me, it looks like it is only looking at `main.go` (you can add `-v` to the `go generate` command to see which files it is looking at). In general, the asset generation is not working correctly for me with your changes. I can make it work by updating the paths (see other comments), but I guess there may be a better way. And with `main.go` moving into `pkg`, this needs update as well I guess.